### PR TITLE
INFRA: Add Build Project

### LINKED ISSUE #10: INFRA: Initialize Infrastructure Build Project

### PR DESCRIPTION
**Pull request by pratikkaje**
Closes #10

### CODE REVIEW

**Comment by pratikkaje** [comment]
Closes #16
Closes #10

File: ExpenseTracker.Core.Infrastructure.Build/Program.cs [hunk 1]
```diff
@@ -1,4 +1,8 @@
-﻿using ADotNet.Clients;
+﻿// -------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE FOR THE WORLD
+// -------------------------------------------------------
+using ADotNet.Clients;
```
Comment (glhays) [inline ExpenseTracker.Core.Infrastructure.Build/Program.cs:6]: add line between Header and using

**Review by glhays — COMMENTED** [review]
Need new yml file for workflow actions

File: ExpenseTracker.Core.Infrastructure.Build/Program.cs (hunk 1 above)
Comment (pratikkaje) [reply ExpenseTracker.Core.Infrastructure.Build/Program.cs:6]: Added the required line break

**Comment by pratikkaje** [comment]
Executed the infra project to get the new yml file.

**Comment by pratikkaje** [comment]
Closes #10 
Closes #15
Closes #16

**Pull request merged**